javascript-loops-practice/includes-seven.js

```diff
@@ -0,0 +1,13 @@
+/* exported includesSeven */
+function includesSeven(array) {
+  var trueOrFalse = false;
+  if (trueOrFalse === false) {
```

Comment:
You don't really need this if statement. You can wait to see if your other if statement hits and return true, or otherwise you can just return false. Here's the instructors solution for this 
<img width="440" alt="Screen Shot 2023-02-02 at 10 00 56 AM" src="https://user-images.githubusercontent.com/71413368/216405400-fc5e24cc-aac6-436f-8580-5cd145858de1.png">